extensions/iata-code-decoder/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# IATA Code Decoder Changelog
+
+## Hello 👋 - (2022-03-20)
```

Comment:
```suggestion
## [Hello 👋] - 2022-03-20
```
We changed the format of the [changelog](https://developers.raycast.com/basics/prepare-an-extension-for-store#version-history)